src/ripple/basics/impl/contract.cpp

```diff
@@ -39,6 +41,18 @@ accessViolation() noexcept
 
 } // detail
 
+void
+LogThrow (std::string title)
+{
+    if (auto j = debugJournal().debug())
+    {
+        j << title;
+        for (auto const& frame : beast::getStackBacktrace ())
+            j << "    " << frame;
+        j << "----";
+    }
+}
```

Comment:
I have another plan for the unhandled exception - it involves `__cxa_throw` and some interesting features of `ld` - it's a bit involved, so will likely need to do it over the weekend.

I'm fine with what you're suggesting by the way – in fact, I think it's a great idea for all the threads under our control.